modules.py

```diff
@@ -210,8 +238,10 @@ class Pupil(OpticalArray):
     """
     Pupil inherits from OpticalArray, it has properties *opd* (the optical path differences) and *name*.
     It can be computed using *_compute_pupil* which uses the opd, or *_path_diff* of the wavefront.
+
+    init takes: wavelength, array_size=1666
     """
-    def __init__(self, wavelength, array_size=101):
+    def __init__(self, wavelength, array_size=1666):
```

Comment:
since 1666 is not reasonable perhaps this should default to a smaller value?  Is the default value here consistent with the default aperture.fits file?